Aztec/Classes/TextKit/ParagraphStyle.swift

```diff
@@ -96,15 +111,29 @@ open class ParagraphStyle: NSMutableParagraphStyle {
         copy.setParagraphStyle(originalCopy)
         copy.textList = textList
         copy.blockquote = blockquote
+        copy.headerLevel = headerLevel
 
         return copy
     }
 
+    open override var hash: Int {
```

Comment:
This is just good practice when you override the isEqual method, this can be used by the system to insert things in Dictionaries so if we have extra properties those should be taken in account.